changes/3d96986df672d56d7143a2a6f91d4ff7.yaml

```diff
@@ -0,0 +1,7 @@
+---
+desc: Add ``uplink:url`` to the output of the ``getCellInfo()`` APIs to indicate which
```

Comment:
I'd prefer we align with the configuration terminology here by using a name like `mirror`